topics/02_Controls/igGrid/11_igGridExcelExporter_Configuring.md

```diff
@@ -0,0 +1,159 @@
+<!--
+|metadata|
+{
+    "fileName": "iggridexcelexporter-configuring",
+    "controlName": "igGridExcelExporter",
+    "tags": [excel, exporting]
+}
+|metadata|
+-->
+
+# Configuring igGridExcelExporter
+
+## Topic Overview
+
+### Purpose
+
+The `igGridExcelExporter`™ provides options that allow any data and layout manipulations caused by the `igGrid` features to be reflected in the Excel Worksheet, as well as to configure styles to be applied, columns to be skipped during exporting, setting file and worksheet names, etc.
+
+### In this topic
+
+This topic contains the following sections:
+
+-   [Configure the `gridFeatureOptions` settings](#gridfeatureoptions_configure)
+-   [Configure exporting igHierarchicalGrid](#hierarchical_data)
+-   [Configure columns that will not be exported](#configure_columns)
+-   [Configure styling](#configure_styling)
+-   [Attach to callbacks (events)](#callbacks)
+-   [Display an overlay while exporting](#exporting_overlay)
+
+### Required background
+- [igGridExcelExporter Overview](iggridexcelexporter-overview.html "igGridExcelExporter Overview") - General information on the `igGridExcelExporter` control.
+
+
+### <a id="gridfeatureoptions_configure"></a>Configure the `gridFeatureOptions` settings
+
+Data and layout manipulations caused by the `Filtering`, `Hiding`, `Paging`, `Sorting`, `Summaries` and `Column Fixing` features of the `igGrid` can be reflected in the exported Excel Worksheet. The setting for each feature is passed within the [`gridFeatureOptions`](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions) object.
+All of these are enumerations and the `sorting`, `summaries` and `columnfixing` have only options - `"none"` or `"applied"`.  
+
+|Setting | Property | Description
+| -------------------| ------------------- |----------- 
+Configure Sorting | [sorting](%%jQueryApiUrl%%/ig.gridexcelexporter#options:settings.gridFeatureOptions.sorting) | Accepts values `"none"` or `"applied"`. When `"none"` Sorting is not applied in the Excel. When `"applied"` Sorting is applied in the Excel Worksheet.
```

Comment:
Please update the reference table for the options that you talk about in this section. Currently it's hard to keep track of what options you talk about and what are the possible values that the developer can set for them.